mleap-spark/src/test/scala/org/apache/spark/ml/parity/classification/MultinomialLogisticRegressionParitySpec.scala

```diff
@@ -0,0 +1,34 @@
+package org.apache.spark.ml.parity.classification
+
+import org.apache.spark.ml.classification.LogisticRegressionModel
+import org.apache.spark.ml.feature.VectorAssembler
+import org.apache.spark.ml.linalg.{Matrices, Vectors}
+import org.apache.spark.ml.parity.SparkParityBase
+import org.apache.spark.ml.{Pipeline, Transformer}
+import org.apache.spark.sql.{DataFrame, Row}
+import org.apache.spark.sql.types.{DoubleType, IntegerType, StructType}
+
+class MultinomialLogisticRegressionParitySpec extends SparkParityBase {
+
+  val labels = Seq(0.0, 1.0, 2.0, 0.0, 1.0, 2.0)
+  val ages = Seq(15, 30, 40, 50, 15, 80)
+  val heights = Seq(175, 190, 155, 160, 170, 180)
+  val weights = Seq(67, 100, 57, 56, 56, 88)
+
+  val rows = spark.sparkContext.parallelize(Seq.tabulate(3) { i => Row(labels(i), ages(i), heights(i), weights(i)) })
+  val schema = new StructType().add("label", DoubleType, nullable = false)
+    .add("age", IntegerType, nullable = false)
+    .add("height", IntegerType, nullable = false)
+    .add("weight", IntegerType, nullable = false)
+
+  override val dataset: DataFrame = spark.sqlContext.createDataFrame(rows, schema)
+
+  override val sparkTransformer: Transformer = new Pipeline().setStages(Array(
+    new VectorAssembler().
+      setInputCols(Array("age", "height", "weight")).
+      setOutputCol("features"),
+    new LogisticRegressionModel(uid = "logr", 
```

Comment:
@hollinwilkins I ended up using LogisticRegressionModel directly for the parity test, because I saw the same in the existing LogisticRegressionParitySpec.scala parity test, which tests the binary case. 

I initially tried to use LogisticRegression instead and as far as I saw, the coefficientMatrix/interceptVector for the LogisticRegressionModel created were the same, but the equalityTest() was failing, because the results were ever so slightly different. Do you happen to know why that would be?

Reply:
@ancasarb I wonder if the values are being stored as JSON and losing some precision? Or losing precision in protobuf?

Reply:
@hollinwilkins the values stored for coefficients/intercepts seemed the same, as far as i could tell. but yea, suspect something similar is happening.